### DISCUSSION TITLE
Parachute freezing players

### DISCUSSION THREAD

**Discussion by vWRENCH** [citizenfx/fivem#2275] [Engineering Discussion]
https://forum.cfx.re/t/parachute-freezing-players/5184994/5

https://forum.cfx.re/t/onesync-crash-with-many-players-in-scope/1609104/4

Will there be any solution? Because my Battle Royale server that holds 1000 players is gradually suffering due to this crash.

**Comment by gottfriedleibniz** [comment]
Another related issue is that AttachDataNode should possibly be referenced in the reassignment/relevant checks (+ maybe other data nodes apply here).

Culled netid's look like they can still leak through cloned task info and some task routines are not well-equipped to handle NULL (or incorrect) entities. Vehicle gadgets look like they could be hit by this as well.
